pkg/tfvars/tfvars.go

```diff
@@ -13,13 +16,19 @@ type config struct {
 	BaseDomain    string `json:"base_domain,omitempty"`
 	MachineCIDR   string `json:"machine_cidr"`
 	Masters       int    `json:"master_count,omitempty"`
+	EtcdPivot     string `json:"etcd_pivot,omitempty"`
 
 	IgnitionBootstrap string `json:"ignition_bootstrap,omitempty"`
 	IgnitionMaster    string `json:"ignition_master,omitempty"`
 }
 
 // TFVars generates terraform.tfvar JSON for launching the cluster.
 func TFVars(clusterID string, clusterDomain string, baseDomain string, machineCIDR *net.IPNet, bootstrapIgn string, masterIgn string, masterCount int) ([]byte, error) {
+	etcdPivot := "false"
```

Comment:
I'm concerned we cannot ship such an env trigger in our release binaries.

Reply:
I believe this is what Alex asked for.
The other thing to note, is that the intention is for this to go away prior to release.
Its purpose is to be an escape hatch until we're all comfortable that we've not broken the world.